src/main/java/seedu/address/model/client/Birthday.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.model.client;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+/**
+ * Represents a client's birthday in MyInsuRec.
+ */
+public class Birthday {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Birthday should be in the format DDMMYYYY";
+    private final LocalDate birthday;
+
+    /**
+     * Constructs a {@code birthdayDate}.
+     *
+     * @param birthday A valid birthday date.
+     */
+    public Birthday(LocalDate birthday) {
+        this.birthday = birthday;
+    }
+
+    @Override
+    public String toString() {
+        if (birthday == null) {
```

Comment:
with the suggestion above, it can help to improve the code here as using `birthday == null` seems a little arbitrary.